framework/docker/da_network_test.go

```diff
@@ -15,8 +15,32 @@ func (s *DockerTestSuite) TestDANetworkCreation() {
 
 	ctx := context.Background()
 
+	// configure different images for different DA node types
+	bridgeNodeConfigs := map[int]*DANodeConfig{
+		0: {
+			Image: &DockerImage{
+				Repository: "ghcr.io/celestiaorg/celestia-node",
+				Version:    "pr-4283",
```

Comment:
for now it's setting the same image as we can't broadcast transactions correctly without the changes in this PR.